tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -492,22 +495,69 @@ def test_generate_messages_with_defaults():
         {
             "event": "data_context.get_batch_list",
             "event_payload": {
-                "batch_request_required_top_level_properties": {
+                "anonymized_batch_request_required_top_level_properties": {
                     "anonymized_datasource_name": "a732a247720783a5931fa7c4606403c2",
                     "anonymized_data_connector_name": "af09acd176f54642635a8a2975305437",
                     "anonymized_data_asset_name": "38b9086d45a8746d014a0d63ad58e331",
                 },
                 "batch_request_optional_top_level_keys": [
-                    "data_connector_query",
                     "batch_spec_passthrough",
+                    "data_connector_query",
                 ],
                 "data_connector_query_keys": ["index"],
                 "runtime_parameters_keys": ["path"],
-                "batch_spec_passthrough_keys": ["reader_options", "reader_method"],
+                "batch_spec_passthrough_keys": ["reader_method", "reader_options"],
             },
             "success": False,
             "version": "1.0.0",
             "event_time": "2020-06-25T16:08:16.030Z",
+            "event_duration": 123,
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.9.manual_testing",
```

Comment:
Can you change to the GE Version where the new message was implemented? It's helpful as a record.

Reply:
Done -- `0.13.45` -- in next commit.